extensions-core/lookups-cached-global/src/main/java/org/apache/druid/server/lookup/namespace/JdbcCacheGenerator.java

```diff
@@ -204,18 +204,20 @@ private Long lastUpdates(CacheScheduler.EntryImpl<JdbcExtractionNamespace> key,
     if (tsColumn == null) {
       return null;
     }
+    final String query = StringUtils.format(
+        "SELECT MAX(%s) FROM %s",
+        tsColumn, table
+    );
     final Timestamp update = dbi.withHandle(
-        handle -> {
-          final String query = StringUtils.format(
-              "SELECT MAX(%s) FROM %s",
-              tsColumn, table
-          );
-          return handle
-              .createQuery(query)
-              .map(TimestampMapper.FIRST)
-              .first();
-        }
+        handle -> handle
+            .createQuery(query)
+            .map(TimestampMapper.FIRST)
+            .first()
     );
+    if (update == null) {
+      LOG.info("Lookup table is empty. No rows returned for the query: %s", query);
```

Comment:
Lets add the look up table name here.

Reply:
@cryptoe The cache scheduler layer doesn't seem to have lookup name information. It only has the JdbcExtractionNamespace level information - which is what gets logged everywhere else in the cache scheduler layer as well.

Technically we can add an extra method param to send lookup name info all the way down to the cache layer, but that seems unnecessary to me. That would also then require us to change logging everywhere else in the cache layer, just to ensure consistency - so it would end up as a much wider change.

Thoughts?

Reply:
You can do this:
```suggestion
      LOG.info("Lookup table[%s] is empty. No rows returned for the query[%s].", table, query);
```

Reply:
@kfaraz Have made the suggested change.

Reply:
Wouldn't INFO logging be too noisy, given that this will log every time we run a check (to see whether to update the table or not)? (By default, it is set to not poll, so the default behaviour is cool)

Reply:
@LakshSingla This is going to be logged only if the table is empty, so I think it should be fine.